### PR TITLE
Payment method demo with docker container

### PR DESCRIPTION
**Pull request by vadim-korsakov**
Payment method demo with docker container

### CODE REVIEW

File: examples/payment-for-all/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Demo shop
```
Comment (jrasanen) [inline examples/payment-for-all/README.md]: Could you write a readme about what this example does?

**Comment by vadim-korsakov** [comment]
Made changes, according to the fix list

**Comment by jrasanen** [comment]
Thanks! Can you change index.html from that "gist" to point to the new directory on github?

File: source/index.html.md [hunk 2]
```diff
@@ -156,7 +156,7 @@ Test credentials
 
 Please note that not all payment methods support testing. The payment methods are enabled that support testing payments. There might be some features that are not working as they are intended to work in the testing side due to limitations to external integrations.
 
-The whole test code is available at: [https://gist.github.com/jrasanen/2a293e7d87443e33184295fab36c96f5](https://gist.github.com/jrasanen/2a293e7d87443e33184295fab36c96f5).
+The whole test code is available at: [https://github.com/vadim-korsakov/checkoutfinland.github.io/tree/master/examples/payment-wall](https://github.com/vadim-korsakov/checkoutfinland.github.io/tree/master/examples/payment-wall).
```
Comment (jrasanen) [inline source/index.html.md]: Could you change this to Checkout Finland's github account?

Comment (jrasanen) [inline source/index.html.md]: Also unit tests need to be fixed, now they check if that gist link is found

**Comment by jrasanen** [comment]
@vadim-korsakov Can you fix the tests, then I think we can merge this. Thanks!

**Pull request closed without merging**